SparseGrids/tsgGridFourier.cpp

```diff
@@ -735,17 +735,29 @@ void GridFourier::evaluateBatchGPUmagma(const double x[], int num_x, double y[],
 }
 
 void GridFourier::integrate(double q[], double *conformal_correction) const{
```

Comment:
* make a separate pull-request, this is a bug that slipped the C++ tests
* check `GridSequence::integrate()` for reference (switch the if-statement)
* there is no need of OpenMP here, `integrate` is something done once and the speedup would be marginal anyway